src/EventListener/DeserializeListener.php

```diff
@@ -52,7 +52,7 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod(Request::METHOD_DELETE)
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
-            || ($request->isMethod(Request::METHOD_PUT) &&  '' === $requestContent)
+            || ($request->isMethod(Request::METHOD_PUT) && '' === $requestContent)
```

Comment:
Ok, I can do it, but it's a little more complicated because it's a "&&" and for a POST we don't initialize the variable. 
So I need to add `$requestContent ?? $request->getContent()` in the deserialize line.

It will be faster but less clear.